src/main/java/codeu/model/store/basic/ActivityStore.java

```diff
@@ -99,18 +98,16 @@ public void addActivityWithoutPersistentStorage(Activity activity) {
         activities.put(activity.getId(), activity);
     }
 
-    /** Find and return the Activity with the given title. */
-//    public Activity getConversationWithType(String title) {
-//        UUID id = titleToId.get(title);
-//        return (id == null) ? null : conversations.get(id);
-//    }
-
     /** Get a list with all the activities. */
     public List<Activity> getActivityList() {
 
         return new ArrayList<>(activities.values());
     }
 
+    public Activity getActivity(UUID id){
+        return (id == null) ? null : activities.get(id);
```

Comment:
Just using `return activities.get(id)` should behave the same way, unless somewhere in the code `null` is used as a key for the map.